### DISCUSSION TITLE
Release ETA?

### DISCUSSION THREAD

**Discussion by Intercity225** [orbat-mapper/orbat-mapper#170] [General]
Is there an ETA for release?

**Comment by kjellmf** [comment]
I'm sorry, but there is no ETA for a release. If there is some specific functionality you need feel free to ask and I may change my priority list.

**Comment by kjellmf** [comment]
I'm working on an updated version 1.0.0 milestone https://github.com/orbat-mapper/orbat-mapper/milestone/1

**Comment by kjellmf** [comment]
I have decided to remove the "beta" label from ORBAT Mapper. Consider it released.
